cloud/pkg/admissioncontroller/admit_ruleendpoint.go

```diff
@@ -3,47 +3,41 @@ package admissioncontroller
 import (
 	"fmt"
 	"net/http"
-	"strings"
 
 	admissionv1beta1 "k8s.io/api/admission/v1beta1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/klog/v2"
 
 	rulesv1 "github.com/kubeedge/kubeedge/cloud/pkg/apis/rules/v1"
 )
 
 func admitRuleEndpoint(review admissionv1beta1.AdmissionReview) *admissionv1beta1.AdmissionResponse {
 	reviewResponse := admissionv1beta1.AdmissionResponse{}
-	var msg string
 	switch review.Request.Operation {
 	case admissionv1beta1.Create:
 		raw := review.Request.Object.Raw
 		ruleEndpoint := rulesv1.RuleEndpoint{}
 		deserializer := codecs.UniversalDeserializer()
 		if _, _, err := deserializer.Decode(raw, nil, &ruleEndpoint); err != nil {
 			klog.Errorf("validation failed with error: %v", err)
-			msg = err.Error()
-			break
+			return toAdmissionResponse(err)
 		}
 		err := validateRuleEndpoint(&ruleEndpoint)
 		if err != nil {
-			msg = err.Error()
-			break
+			return toAdmissionResponse(err)
 		}
 		reviewResponse.Allowed = true
 		klog.Info("admission validation passed!")
```

Comment:
Is this `info log` necessary?
If it is not useful, can you consider deleting it?

Reply:
fixed.